couchbase/query_options.hxx

```diff
@@ -567,5 +568,5 @@ struct query_options : public common_options<query_options> {
  * @since 1.0.0
  * @uncommitted
  */
-using query_handler = std::function<void(couchbase::query_error_context, query_result)>;
+using query_handler = std::function<void(couchbase::error, query_result)>;
```

Comment:
NIT: I think we could just use `error` without the full namespace since we're already inside the `couchbase` namespace.